src/magicbox/SuggestionsManager.ts

```diff
@@ -439,8 +389,7 @@ export class SuggestionsManager {
     input.setAttribute('aria-autocomplete', 'list');
   }
 
-  private htmlElementIsSuggestion(selected: HTMLElement) {
-    const omniboxSelectables = $$(this.element).findAll(`.${this.options.suggestionClass}`);
-    return indexOf(omniboxSelectables, selected) > -1;
+  private getSuggestionFromElement(element: HTMLElement) {
+    return find(this.displayedSuggestions, suggestion => suggestion.dom === element);
```

Comment:
I find it a bit confusing that `dom` is of type HTMLElement when the Dom type is also commonly used in many places

Reply:
I agree. Should it be renamed in this PR or be part of [JSUI-2700](https://coveord.atlassian.net/browse/JSUI-2700)?